completions/quick-lint-js.bash

```diff
@@ -1,11 +1,15 @@
 # Copyright (C) 2020  Matthew "strager" Glazar
 # See end of file for extended copyright information.
 
+_quick_lint_js_output_formats="gnu-like vim-qflist-json emacs-lisp"
+_quick_lint_js_diagnostic_hyperlink_modes="auto always never"
+_quick_lint_js_languages="default javascript javascript-jsx experimental-typescript experimental-typescript-jsx"
+
 _quick-lint-js () {
         local cur opts
         _init_completion -n = || return
 
-        opts='--help --version --lsp-server --config-file= --stdin --exit-fail-on= --output-format= --diagnostic-hyperlinks= --vim-file-bufnr='
+        opts="--help --version --lsp --lsp-server --config-file= --stdin --exit-fail-on= --output-format= --diagnostic-hyperlinks= --vim-file-bufnr= --language= --path-for-config-search= --snarky --debug-apps"
```

Comment:
`--lsp` and `--lsp-server` are the same. (`--lsp` exists only for people who guess command-line options.) I think we should show complete `--lsp-server`.

Reply:
just to confirm, do we wanna remove `--lsp` from shell completion?